test/language/computed-property-names/computed-property-names-setter-class.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 12.2.5
+description: >
+    In a class, duplicate computed property setter names produce only a single property of
```

Comment:
Fixed in next push. (There are now two files each, for getter and setter, one that has a single case in it and one for duplicate keys)